src/_scss/pages/recipient/_contentWrap.scss

```diff
@@ -140,6 +138,10 @@
 					}
 				}
 			}
+
+			.recipient-section__viz.details {
+				margin-left: 0 !important;
```

Comment:
is there a way we could move this style, so we don't have to use the !important?  especially since the recipient-section__viz class should be in this file or a closely related file

Reply:
i must not have refreshed my browser before turns out it was unnecessary